### PR TITLE
Revert PR_9884 “Migrate HtmlHelp to CsWin32” and add unit test

### PR DESCRIPTION
**Pull request by Epica3055**
Fixes https://github.com/dotnet/winforms/issues/10410

Related to [PR_9884](https://github.com/dotnet/winforms/pull/9884)

The issue resulted from  [PR_9884](https://github.com/dotnet/winforms/pull/9884)

# Proposed changes
- Revert PR
- Add unit test

### CODE REVIEW

**Comment by lonitra** [comment]
To confirm, was this issue with the metadata?

File: src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/Help/HelpTests.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Drawing;
+
+namespace System.Windows.Forms.Tests;
+
+public class HelpTests
+{
+    [Fact]
+    public void ShowHelpTest()
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/Help/HelpTests.cs:11]: @Epica3055  - this is the minimal test. Could you please add more coverage?

**Comment by Tanya-Solyanik** [comment]
>To confirm, was this issue with the metadata?

Yes, metadata is already fixed, thanks to Lachlan, do we know how much time it would take to update metadata and Win32?

**Comment by elachlan** [comment]
Can happen in a day or so depending on the teams.

File: src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/Help/HelpTests.cs (hunk 1 above)
Comment (Epica3055) [reply src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/Help/HelpTests.cs]: for example when I write one like this
``` c#
    [Fact]
    public void ShowHelpTest()
    {
        Help.ShowHelp(null, "https://www.microsoft.com/");
    }
```
it doesn't involve using native dll.

Comment (Tanya-Solyanik) [reply src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/Help/HelpTests.cs]: I see, you are right, for this issue, one is enough. However we need code coverage for all methods on this control eventually. I thought that once you are working in this space, it's easy to add more tests.

**Comment by Tanya-Solyanik** [comment]
If the new metadata is not [published ](https://github.com/microsoft/win32metadata/issues/1749)by the middle of the next week, we'll merge this PR and re-open https://github.com/dotnet/winforms/pull/9884

**Comment by Epica3055** [comment]
> If the new metadata is not [published ](https://github.com/microsoft/win32metadata/issues/1749)by the middle of the next week, we'll merge this PR and re-open #9884

@Tanya-Solyanik yeah, no problem.

**Review by lonitra — APPROVED** [review]

File: src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/Help/HelpTests.cs (hunk 1 above)
Comment (Epica3055) [reply src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/Help/HelpTests.cs]: This `Help` is like a general method, not for a specific Control.

https://github.com/dotnet/winforms/assets/45864985/9e6df513-d035-4749-bfae-ed86cc30d900

I guess [this issue](https://github.com/dotnet/winforms/issues/10453) can  track all Control tests.

**Pull request merged**